test/ecto/adapters/postgres_test.exs

```diff
@@ -919,8 +920,8 @@ defmodule Ecto.Adapters.PostgresTest do
 
   test "alter table with prefix" do
     alter = {:alter, table(:posts, prefix: :foo),
-             [{:add, :author_id, references(:author, prefix: :foo), []},
-              {:modify, :permalink_id, references(:permalinks, prefix: :foo), null: false}]}
+             [{:add, :author_id, %Reference{table: :author}, []},
```

Comment:
@josevalim, I have removed the prefix here, since Reference doesn't have it and it wouldn't be set anyway. (same thing for mysql adapter test.